### DISCUSSION TITLE
Cycle option inconsistent behaviour

### DISCUSSION THREAD

**Discussion by gab3d** [opentoonz/opentoonz#6807] [Ideas]
### Issue Summary
When applying the Cycle option to cycle certain animation channels, the behaviour is not the same if you apply it from the Xsheet and from the Function Editor.

### Steps to Reproduce
1. Create any keyframe animation on a level, using the Animate tool.
2. Apply the Cycle option from the Xsheet or from the Function Editor graph editor.

### Expected Results
Cycled keyframes to behave in the same manner.
If I'm asked, "Relative" is what I see as most useful, but I won't mind having both "Relative" and "Absolute" modes (among others as well, perhaps "Oscilate" could be a good addition too...)

### Actual Results
A. Cycled channels activated from the Xsheet, cycle in an "absolute" way (i.e. each new cycle starts again from the exact values as in the first keyframe of the cycle)
B. Cycled channels activated from the Function Editor's graph editor, cycle in a relative way (i.e. each new cycle starts from the value of the last keyframe and displaces from that point on).

### System Information
* **OpenToonz Version**:  latest nightly

**Comment by RodneyBaker** [comment]
As a lower priority this enhancement appears to be slipping to the 2019 development cycle.

**Comment by RodneyBaker** [comment]
Transferring this to Discussions so the desired way forward can be determined.
